index.html

```diff
@@ -8,44 +8,273 @@
+                        <a class="supporting-text"">LET'S CONNECT</a>
+                    <ul class="about-social-media-links">
+                        <li>
+                            <a href="#">
+                                <img src="./img/twitter.png" alt="twitter link">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#">
+                                <img src="./img/Linkedin.png" alt="Linkedin link">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#">
+                                <img src="./img/m.png" alt="m link">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#">
+                                <img src="./img/github.png" alt="twitter link">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#">
+                                <img src="./img/m2.png" alt="m2 link">
+                            </a>
+                        </li>
+                    </ul>
+                </div>
+                <button id="btn-get-resume"><span> Get my resume</span></button>
+            </div>
+            <ul class="language-and-skills-frame">
+                <li id="skills">
+                    <div class="languages-frame">
+                        <div class="languages-frame-title">
+                            <img src="img/Disabled.png" alt="" class="languages-frame-dropdown">
+                            <h3 class="languages-frame-title-h3"> Languages</h3>
+                        </div>
+                        <ul class="language-list">
+                            <li> 
+                                <div class="icon-language icon-language-javascript"></div>
+                                <p>JavaScript</p>
+                            </li>
+                            <li> 
+                                <div class="icon-language icon-language-html"></div>
+                                <p>HTML</p>
+                            </li>
+                            <li> 
+                                <div class="icon-language icon-language-css"></div>
+                                <p>CSS</p>
+                            </li>
+                        </ul>
+                    </div>
                 </li>
-                <li>
-                    <a href="#">
-                        <img src="./img/github.png" alt="twitter link">
-                    </a>
+                <li class="frameworks">
+                    <div class="frameworks-frame-title">
+                        <img src="img/Disabled.png" alt="" class="framework-frame-dropdown">
+                        <h3 class="frameworks-frame-title-h3"> Frameworks</h3>
+                    </div>
                 </li>
```

Comment:
- [Optional] Please remove the spacing between the list of technologies to match the figma template. You are expected to stick to design as much as possible.
### Your design
![12](https://user-images.githubusercontent.com/94551484/192089902-accbd4d0-c3eb-43af-aa5e-e9e5f62d151a.png)
### Figma template
![f12](https://user-images.githubusercontent.com/94551484/192089905-8c86d452-8f87-4dc9-8f93-d72238a82d8a.png)